payload-dependencies/pom.xml

```diff
@@ -62,6 +62,10 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>scm-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins</groupId>
+      <artifactId>configuration-as-code</artifactId>
```

Comment:
It makes JCasC a mandatory requirement for all Jenkinsfile Runner instances. I do not mind.

Reply:
Yeah, I couldn't come up with a better approach, sadly.

Reply:
Not a problem IMHO. We can keep it mandatory for now, and revisit if it causes issues.